socket-io/build.gradle.kts

```diff
@@ -6,23 +6,51 @@ import java.util.Base64
 import kotlin.text.String
 
 plugins {
-    plugin(Deps.Plugins.androidLibrary)
-    plugin(Deps.Plugins.kotlinMultiplatform)
-    plugin(Deps.Plugins.kotlinAndroidExtensions)
-    plugin(Deps.Plugins.mobileMultiplatform)
-    plugin(Deps.Plugins.mavenPublish)
-    plugin(Deps.Plugins.signing)
+    id("com.android.library")
+    id("org.jetbrains.kotlin.multiplatform")
+    id("kotlin-android-extensions")
+    id("dev.icerock.mobile.multiplatform")
+    id("org.gradle.maven-publish")
+    id("signing")
 }
 
 group = "dev.icerock.moko"
-version = Deps.mokoSocketIoVersion
+version = libs.versions.mokoSocketIoVersion.get()
+
+kotlin {
+    jvm()
+
+    sourceSets {
+        val commonMain by getting
+
+        val commonJvm = create("commonJvm") {
+            dependsOn(commonMain)
+            dependencies {
+                implementation(libs.appCompat)
+                implementation(libs.socketIo)
+            }
+        }
+
+        val androidMain by getting {
+            dependsOn(commonJvm)
+        }
+
+        val jvmMain by getting {
+            dependsOn(commonJvm)
+        }
+
+    }
+}
 
 dependencies {
-    commonMainImplementation(Deps.Libs.MultiPlatform.serialization)
+    commonMainImplementation(libs.serialization)
+}
 
-    androidMainImplementation(Deps.Libs.Android.appCompat)
-    androidMainImplementation(Deps.Libs.Android.socketIo) {
-        exclude(group = "org.json", module = "json")
+configurations.all {
+    resolutionStrategy.eachDependency {
+        if (requested.name == "socket.io-client") {
+            exclude(group = "org.json", module = "json")
+        }
     }
```

Comment:
not needed if pass excluding inside dependency declaration

Reply:
and need to check runtime work on jvm - i think without this artifact jvm will broke. if so - we should add json dependency for `jvmMainImplementation`